.github/workflows/ci.yml

```diff
@@ -82,7 +82,14 @@ jobs:
       - name: Install dependencies
         run: tools/flutter-dependencies.sh
       - name: Run tests
-        run: (cd packages/flutter && flutter test)
+        run: (cd packages/flutter && flutter test --coverage)
+      - name: Upload code coverage
+        uses: codecov/codecov-action@v2
+        if: ${{ always() }}
+        with:
+          files: packages/flutter/coverage/lcov.info
```

Comment:
can't you specify a common file for both outputs and let the plug-in handle the merging?
other maybe you could try to specific both paths to both packages in both septs, comma separated?

Reply:
I think codecov can't read the json format. Let's see if this works, first.